api/src/main/java/com/redhat/ipaas/api/v1/model/ConnectorGroup.java

```diff
@@ -21,30 +21,30 @@
 import java.io.Serializable;
 
 /**
- * ComponentGroups are labels in Camel.
+ * ConnectorGroups are labels in Camel.
  *
  * https://github.com/apache/camel/blob/master/platforms/catalog/src/main/java/org/apache/camel/catalog/CamelCatalog.java#L233
```

Comment:
This link is no longer valid. Please link to commit rather than master if we think this is useful (it probably is).